src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -59,6 +60,8 @@ public class SnowflakeSinkConnector extends SinkConnector {
   // Using setupComplete to synchronize
   private boolean setupComplete;
 
+  private static final long VALIDATION_NETWORK_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(45);
```

Comment:
I thought you have added a few timeout values in another PR, what's the difference between this and the others?

Reply:
This timeout is for JDBC connection. https://docs.snowflake.com/en/user-guide/jdbc-parameters.html#networktimeout
Others were for httpclient which is used to call Snowpipe REST APIs.